src/library/searchquery.cpp

```diff
@@ -166,6 +172,46 @@ QString TextFilterNode::toSql() const {
     return concatSqlClauses(searchClauses, "OR");
 }
 
+CrateFilterNode::CrateFilterNode(const QSqlDatabase& database,
+                                 const QString& argument)
+    : m_database(database),
+      m_argument(argument) {
+    getTrackIds();
+}
+
+void CrateFilterNode::getTrackIds() {
+    FieldEscaper escaper(m_database);
+    QString escapedArgument = escaper.escapeString(kSqlLikeMatchAll + m_argument + kSqlLikeMatchAll);
```

Comment:
Only escape the argument, not the LIKE wildcards:
kSqlLikeMatchAll + FieldEscaper(m_database).escape(m_argument) + kSqlLikeMatchAll
The result might be equal in this special case, but generally we don't want to escape special SQL characters that are part of the query statement and not part of any parameter.